frontend/src/app/admin/page.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+import Dashboard from "@/components/dashboard/dashboard";
+import Navbar from "@/components/navbar/navbar";
+import { useEffect, useState } from "react";
+
+export default function Home() {
+  const [name, setName] = useState<string>("");
+  useEffect(() => {
+    const storedName = localStorage.getItem("name")
+      ? String(localStorage.getItem("name"))
+      : "";
+    setName(storedName);
+  }, []);
+
+  return (
+    <div>
+      <Navbar name={name} />
+      <Dashboard />
+    </div>
+  );
+}
```

Comment:
I felt like wanting to know the user's email or id might be useful but i guess it's not a big deal

Reply:
I think `id` is fine to store if we need it, but I didn't see it used anywhere so I did not store it in this PR.
We can always get the user's email through `GET /api/user/{id}`, so I think we should avoid putting it in insecure storage just to avoid the extra request.

Reply:
ok